CI test fails: DuckDuckGoSearchException: https://lite.duckduckgo.com/lite/ 202 Ratelimit

CI test fails:
> DuckDuckGoSearchException: https://lite.duckduckgo.com/lite/ 202 Ratelimit

See: https://github.com/huggingface/smolagents/actions/runs/15002040575/job/42151265147?pr=1322
```
FAILED tests/test_default_tools.py::DefaultToolTests::test_ddgs_with_kwargs - duckduckgo_search.exceptions.DuckDuckGoSearchException: https://lite.duckduckgo.com/lite/ 202 Ratelimit
FAILED tests/test_search.py::TestDuckDuckGoSearchTool::test_agent_type_output - duckduckgo_search.exceptions.DuckDuckGoSearchException: https://lite.duckduckgo.com/lite/ 202 Ratelimit
```

--- a/tests/test_default_tools.py
+++ b/tests/test_default_tools.py
@@ -26,6 +26,7 @@ from smolagents.default_tools import (
 )
 
 from .test_tools import ToolTesterMixin
+from .utils.markers import require_run_all
 
 
 class DefaultToolTests(unittest.TestCase):
@@ -35,6 +36,7 @@ class DefaultToolTests(unittest.TestCase):
         assert isinstance(result, str)
         assert "* [About Wikipedia](/wiki/Wikipedia:About)" in result  # Proper wikipedia pages have an About
 
+    @require_run_all
     def test_ddgs_with_kwargs(self):
         result = DuckDuckGoSearchTool(timeout=20)("DeepSeek parent company")
         assert isinstance(result, str)

--- a/tests/test_search.py
+++ b/tests/test_search.py
@@ -29,3 +29,7 @@ class TestDuckDuckGoSearchTool(ToolTesterMixin):
     def test_exact_match_arg(self):
         result = self.tool("Agents")
         assert isinstance(result, str)
+
+    @require_run_all
+    def test_agent_type_output(self):
+        super().test_agent_type_output()

